swift-6-beta.docc/LanguageGuide/OpaqueTypes.md

```diff
@@ -1,41 +1,16 @@
-<!--
-要翻译的文件：https://github.com/SwiftGGTeam/the-swift-programming-language-in-chinese/blob/swift-6-beta-translation/swift-6-beta.docc/LanguageGuide/OpaqueTypes.md
-Swift 文档源文件地址：https://docs.swift.org/swift-book/documentation/the-swift-programming-language/opaquetypes
-翻译估计用时：⭐️⭐️⭐️⭐️⭐️
--->
-
-# Opaque and Boxed Protocol Types
+# 不透明类型和封装协议类型
 
-Hide implementation details about a value's type.
+隐藏关于值类型的实现细节。
 
-Swift provides two ways to hide details about a value's type:
-opaque types and boxed protocol types.
-Hiding type information
-is useful at boundaries between
-a module and code that calls into the module,
-because the underlying type of the return value can remain private.
+Swift 提供了两种隐藏值类型细节的方法：不透明类型（Opaque Type）和封装协议类型（Boxed Protocol Type）。在模块与模块调用端之间的边界上隐藏类型信息是有用的，因为这样返回值的底层类型可以保持为私有的可访问状态。
 
-A function or method that returns an opaque type
-hides its return value's type information.
-Instead of providing a concrete type as the function's return type,
-the return value is described in terms of the protocols it supports.
-Opaque types preserve type identity ---
-the compiler has access to the type information,
-but clients of the module don't.
+返回不透明类型的函数或方法隐藏了其返回值的类型信息。相较于提供一个具体的类型作为函数的返回类型，它会根据它支持的协议来描述它的返回值。不透明类型会保留类型的身份信息 —— 编译器可以访问该类型信息，但模块的调用端则无法访问。
```

Comment:
> Instead of providing a concrete type as the function's return type,
 the return value is described in terms of the protocols it supports.

函数会将返回值类型描述为一个符合某种协议的类型，而非一个更具体的类型。